### PR TITLE
Update documentation with info about job script collection.

### PR DESCRIPTION
**Pull request by jpwhite4**
This documentation describes the config settings avialable in the 1.4
release of the supremm summarization software.

See also https://github.com/ubccr/supremm/pull/217

These docs will go into the 9.0 version. The 1.4 release of the summarization s/w will happen at the same time  as the 9.0 (or perhaps slightly before).

### CODE REVIEW

File: docs/supremm-jobscript.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+The job viewer tab in XDMoD is able to show the job batch script along
```
Comment (jtpalmer) [inline docs/supremm-jobscript.md]: I've been trying to always use "Open XDMoD" and not "XDMoD" for branding purposes.  Do you think it's worth doing that everywhere for consistency?

File: docs/supremm-jobscript.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+with the other job information. The job batch scripts are stored in the
+database using the `ingest_jobscripts.py` script. This script is
+packaged with the [Job Summarization software](supremm-processing-install.md).
+
+The job script ingestion is an optional feature. It is _not_ required for XDMoD
```
Comment (jtpalmer) [inline docs/supremm-jobscript.md]:
```suggestion
Job batch script ingestion is an optional feature. It is _not_ required for XDMoD
```

or maybe "The ingestion of job batch scripts ..."

File: docs/supremm-jobscript.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+to display job performance information.
+
+The mechanism for extracting the job batch script is resource manager specific. Consult the
+documentation for your resource manager software for information about
+how to enable logging of the batch script.
```
Comment (jtpalmer) [inline docs/supremm-jobscript.md]:
```suggestion
how to enable logging of the job batch scripts.
```

File: docs/supremm-jobscript.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+## Source data schema
+
+In order to ingest the script data into XDMoD the job scripts must be stored in files. There
+must be one file per job and the jobs for a given day should be stored in a
```
Comment (jtpalmer) [inline docs/supremm-jobscript.md]:
```suggestion
must be one file per job and the job batch scripts for a given day should be stored in a
```

File: docs/supremm-jobscript.md (hunk 1 above)
Comment (jpwhite4) [reply docs/supremm-jobscript.md]: Yep

File: docs/_data/toc.yml [hunk 5]
```diff
@@ -31,6 +31,8 @@ toc:
     subfolderitems:
       - page: PCP Data collection
         url: /supremm-compute-pcp.html
+      - page: Job Script collection
```
Comment (jtpalmer) [inline docs/_data/toc.yml]: May as well change this one too:
```suggestion
      - page: Job Batch Script collection
```

**Pull request merged**